### DISCUSSION TITLE
falling lines

### DISCUSSION THREAD

**Discussion by Anubias25** [tannerhelland/PhotoDemon#439] [General]
Please help: Where can I find a way to correct converging lines in Photodemon?

**Comment by tannerhelland** [comment]
Hello @Anubias25 .  The easiest way to fix converging lines in PhotoDemon is with the `Effects > Transform > Perspective` tool.  

Hope that helps!

**Comment by Anubias25** [comment]
Thanks! Very good answer. I really hadn't found it myself. program is great!
